src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx

```diff
@@ -193,23 +191,10 @@ export default function AppointmentsPage() {
 
   const history = useHistory();
 
-  let paragraphText = 'These appointment requests haven’t been scheduled yet.';
-  if (featureStatusImprovement) {
-    paragraphText =
-      'Your appointment requests that haven’t been scheduled yet.';
-  }
   return (
     <PageLayout showBreadcrumbs showNeedHelp>
-      <h1 className={getSpacing({ isPrintList, isPast, isPending })}>
-        {pageTitle}
-      </h1>
+      <h1 className={getSpacing({ isPast, isPending })}>{pageTitle}</h1>
```

Comment:
Once `getSpacing` function is eliminated , replace the call with the actual classnames

Reply:
Done.